providers/aws/services/vpc/vpc_service_test.py

```diff
@@ -0,0 +1,254 @@
+
+    # Test VPC Describe VPC Peering connections
+    @mock_ec2
+    def test__describe_route_tables__(self):
+        # Generate VPC Client
+        ec2_client = client("ec2", region_name=AWS_REGION)
+        ec2_resource = resource("ec2", region_name=AWS_REGION)
+
+        # Create VPCs peers as well as a route
+        vpc = ec2_client.create_vpc(CidrBlock="10.0.0.0/16")
+        peer_vpc = ec2_client.create_vpc(CidrBlock="11.0.0.0/16")
+        vpc_pcx = ec2_client.create_vpc_peering_connection(
+            VpcId=vpc["Vpc"]["VpcId"], PeerVpcId=peer_vpc["Vpc"]["VpcId"]
+        )
+        vpc_pcx_id = vpc_pcx["VpcPeeringConnection"]["VpcPeeringConnectionId"]
+
+        vpc_pcx = ec2_client.accept_vpc_peering_connection(
+            VpcPeeringConnectionId=vpc_pcx_id
+        )
+        main_route_table_id = ec2_client.describe_route_tables(
+            Filters=[
+                {"Name": "vpc-id", "Values": [vpc["Vpc"]["VpcId"]]},
+                {"Name": "association.main", "Values": ["true"]},
+            ]
+        )["RouteTables"][0]["RouteTableId"]
+        # FilterNotImplementedError: The filter 'route.vpc-peering-connection-id' for DescribeRouteTables has not been implemented in Moto yet.
+        # main_route_table = ec2_resource.RouteTable(main_route_table_id)
+        # main_route_table.create_route(
+        #     DestinationCidrBlock="10.0.0.4/24", VpcPeeringConnectionId=vpc_pcx_id
+        # )
+
+        # VPC client for this test class
+        audit_info = self.set_mocked_audit_info()
+        vpc = VPC(audit_info)
+        vpc.vpc_peering_connections[0].route_tables = [
+            Route(
+                main_route_table_id,
+                ["10.0.0.4/24"],
+            )
+        ]
+        assert len(vpc.vpc_peering_connections[0].route_tables) == 1
+
+    # Test VPC Describe VPC Endpoints
+    @mock_ec2
+    def test__describe_vpc_endpoints__(self):
+        # Generate VPC Client
+        ec2_client = client("ec2", region_name=AWS_REGION)
+        # Create VPC endpoint
+        vpc = ec2_client.create_vpc(CidrBlock="10.0.0.0/16")["Vpc"]
+
+        route_table = ec2_client.create_route_table(VpcId=vpc["VpcId"])["RouteTable"]
+        ec2_client.create_vpc_endpoint(
+            VpcId=vpc["VpcId"],
+            ServiceName="com.amazonaws.us-east-1.s3",
+            RouteTableIds=[route_table["RouteTableId"]],
+            VpcEndpointType="Gateway",
+            PolicyDocument=json.dumps(
+                {
+                    "Statement": [
+                        {
+                            "Action": "*",
+                            "Effect": "Allow",
+                            "Principal": "*",
+                            "Resource": "*",
+                        }
+                    ]
+                }
+            ),
+        )
+        # VPC client for this test class
+        audit_info = self.set_mocked_audit_info()
+        vpc = VPC(audit_info)
+        assert len(vpc.vpc_endpoints) == 1
```

Comment:
Also check that the values created are the same that the received in the client

Reply:
Done